src/Button/Button.tsx

```diff
@@ -6,33 +6,33 @@ import ButtonBase, {ButtonBaseProps, ButtonSystemProps, buttonSystemProps} from
 import {ComponentProps} from '../utils/types'
 
 const Button = styled(ButtonBase)<ButtonBaseProps & ButtonSystemProps & SxProp>`
-  color: ${get('buttons.default.color.default')};
-  background-color: ${get('buttons.default.bg.default')};
-  border: 1px solid ${get('buttons.default.border.default')};
-  box-shadow: ${get('buttons.default.shadow.default')};
+  color: ${get('colors.btn.text')};
+  background-color: ${get('colors.btn.bg')};
+  border: 1px solid ${get('colors.btn.border')};
+  box-shadow: ${get('shadows.btn.shadow')}, ${get('shadows.btn.insetShadow')}};
 
   &:hover {
-    background-color: ${get('buttons.default.bg.hover')};
-    border-color: ${get('buttons.default.border.hover')};
-    box-shadow: ${get('buttons.default.shadow.hover')};
+    background-color: ${get('colors.btn.hoverBg')};
+    border-color: ${get('colors.btn.hoverBorder')};
+    box-shadow: ${get('shadows.btn.focusShadow')}, ${get('shadows.btn.insetShadow')};
```

Comment:
There didn't seem to be a clear mapping here for hover shadow in primer css, took my best guess 🤷‍♂️